incubator/kafka/templates/service-brokers-external.yaml

```diff
@@ -21,12 +31,18 @@ metadata:
     heritage: {{ $root.Release.Service }}
     pod: {{ $responsiblePod | quote }}
 spec:
-  type: NodePort
+  type: {{ $root.Values.external.type }}
   ports:
     - name: external-broker
+      {{- if eq $root.Values.external.type "LoadBalancer" }}
```

Comment:
make spacing consistent on this block of `if/else/end`

Reply:
I've updated this to be consistent throughout the file.  I am unsure if I have addressed your comment.

Reply:
Thanks @drekle